cmd/newrelic/command_decode.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"encoding/base64"
+	"fmt"
+	"net/url"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/tidwall/gjson"
+)
+
+// Command represents the decode command.
+var cmdDecode = &cobra.Command{
+	Use:     "decode",
+	Short:   "Decodes NR1 URL Strings ",
+	Example: `newrelic decode https://one.newrelic.com/launcher/nr1-core.home?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5ob21lLXNjcmVlbiJ9&platform[accountId]=1`,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		relicString := strings.Join(args, " ")
+		u, _ := url.Parse(relicString)
+		relicString = u.Query().Get("pane")
+		decoded, err := base64.StdEncoding.DecodeString(relicString)
+
+		if err != nil {
+			fmt.Println("decode error:", err)
+		}
+
+		decodedURL := string(decoded)
+		fmt.Println("\n**** Decoded URL **** ")
+		fmt.Println(decodedURL)
+
+		entityID := strings.Contains(decodedURL, "entityId")
+
+		if entityID {
+			var regex = regexp.MustCompile(`[a-zA-Z0-9\+]*={0,3}`)
+			entityIDString := gjson.Get(decodedURL, "entityId")
+			decodedURL = entityIDString.String()
+
+			if len(regex.FindStringIndex(decodedURL)) > 0 {
+				for (len(decodedURL) % 4) != 0 {
+					decodedURL += "="
+				}
+			}
+
+			decoded, err := base64.StdEncoding.DecodeString(decodedURL)
+			if err != nil {
+				fmt.Println("decode error:", err)
+			}
+
+			decodedURL := string(decoded)
+			fmt.Println("\n**** Decoded Entity ID **** ")
```

Comment:
Awesome stuff to get this going. I think we can move forward with this, but we might want to adjust the command signature to be a little more declarative. A declarative signature will help us avoid needing the extra line of output to describe what we're displaying. 

Hypothetical command signature:
```zsh
newrelic decode url https://yadda.yadda.com?pane=eyJuZXJk... --param "pane" --search "entityId"
```

This signature allows the user to define what they want from the URL's query param `pane` as well as what to search for in the encoded string. The `--query-param` flag value can then be passed to the function on [line 23](https://github.com/newrelic/newrelic-cli/pull/653/files#diff-90f45227f18dce8728ec5764cfbef896385ae0d17ac97f9b37ff5d6b64bd2c18R23) and the `--search` value can be passed to [line 34](https://github.com/newrelic/newrelic-cli/pull/653/files#diff-90f45227f18dce8728ec5764cfbef896385ae0d17ac97f9b37ff5d6b64bd2c18R34) and [line 38](https://github.com/newrelic/newrelic-cli/pull/653/files#diff-90f45227f18dce8728ec5764cfbef896385ae0d17ac97f9b37ff5d6b64bd2c18R38) (`gjson.Get(decodedURL, decodeSearch)`) or something to that effect. 

This give the user the ability to decode other URL parts as well if/when things change down the line.